.github/workflows/pending-review-notification.yml

```diff
@@ -3,6 +3,10 @@ on:
   schedule:
     # Every Tuesday and Thursday at 00:00 UTC.
     - cron: '0 0 * * 2,5'
+  # Github doesn't assure running the scheduled jobs on time,
```

Comment:
GitHub doesn't provide assurance that the scheduled jobs will run on time (see https://....). So, we add the workflow_dispatch event here to allow triggering this workflow manually if needed.